dev-util/gdb/patches/gdb-15.1.patchset

```diff
@@ -0,0 +1,7181 @@
+From 9c9cd5cf820ea4f83b72409dcaaa797f351d2ea1 Mon Sep 17 00:00:00 2001
+From: Trung Nguyen <me@trungnt2910.com>
+Date: Mon, 29 Jul 2024 22:55:42 +1000
+Subject: [PATCH 1/3] gdb/remote: fix assertion failure during startup
```

Comment:
What's the link for the upstream report for this patch?

Reply:
https://sourceware.org/pipermail/gdb-patches/2024-July/210771.html